packages/app/config/webpack.config.prod.js

```diff
@@ -59,7 +59,7 @@ const extractTextPluginOptions = shouldUseRelativeAssetPaths
       { publicPath: Array(cssFilename.split('/').length).join('../') }
     : {};
 
-const scriptPrefix = '..';
+const scriptPrefix = '../../..';
```

Comment:
Instead of hardcoding this, it can probably be picked up from the manifest activities.dhis.href property